pkgs/os-specific/linux/metastore/default.nix

```
@@ -0,0 +1,23 @@
+{ stdenv, libbsd, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  version = "1.1.2";
+  pname = "metastore";
```

please add new line between pname and src.
```suggestion
  pname = "metastore";

```

---

I made the change, but your comment seems to stay there regardless